runnumber-rest/rest.py

```diff
@@ -1,4 +1,25 @@
+<<<<<<< HEAD
```

Comment:
Looks like the merge failed and left behind artifacts

Reply:
This was a simple forgot to save changes before committing